tests/Aspire.Hosting.Tests/ExpressionResolverTests.cs

```diff
@@ -39,7 +39,7 @@ public async Task ResolveInternalAsync_ResolvesCorrectly(ExpressionResolverTestD
         data.Add(new ExpressionResolverTestData(true, new ConnectionStringReference(new TestExpressionResolverResource("SecretParameter"), false)), null, ("SecretParameter", true));
 
         // IResourceWithConnectionString resolves differently for ResourceWithConnectionStringSurrogate (as a secret parameter)
```

Comment:
```suggestion
        // IResourceWithConnectionString resolves differently for ConnectionStringParameterResource (as a secret parameter)
```

Reply:
There are a few references to `ResourceWithConnectionStringSurrogate` left in the code. Can you find and replace them?

Reply:
Resolved